pkg/common/configs/config_test.go

```diff
@@ -916,35 +916,35 @@ partitions:
 
 	// limit 1 check
 	limit := conf.Partitions[0].Limits[0]
-	if len(limit.Users) != 1 && limit.Users[0] != "user1" && len(limit.Groups) != 0 {
+	if len(limit.Users) != 1 || limit.Users[0] != "user1" || len(limit.Groups) != 0 {
 		t.Errorf("failed to load max apps %v", limit)
 	}
 	if len(limit.Groups) != 0 {
 		t.Errorf("group list should be empty and is not: %v", limit)
 	}
-	if limit.MaxApplications != 10 && (len(limit.MaxResources) != 2 && limit.MaxResources["memory"] != "10000") {
+	if limit.MaxApplications != 5 || len(limit.MaxResources) != 2 || limit.MaxResources["memory"] != "10000" {
```

Comment:
Do we want to add one more condition `limit.MaxResources["vcore"] != "10"`?

Reply:
Thanks for the comments !